src/components/common/Tile/Tile.jsx

```diff
@@ -1,59 +1,73 @@
 import {
-  Container,
-  Content,
-  Header,
-  Image,
-  Year,
-  Release,
-  ReleaseDate,
-  GreyRelease,
-  Genres,
-  Genre,
-  Rating,
-  IconStar,
-  Rate,
-  Votes,
-  Discription,
-  ContainerRateText,
-} from "./Styled";
+	Container,
+	Content,
+	Title,
+	Image,
+	Year,
+	Release,
+	Genres,
+	Genre,
+	Rating,
+	IconStar,
+	Rate,
+	Votes,
+	Discription,
+	ImageWrapper,
+	Production,
+	OutOf,
+	ReleaseDate
+} from "./styled";
 
-export const Tile = ({ poster_path, title, release_date, production_countries, vote_average, votes, overview, genres }) => {
-  const isMobile = window.innerWidth < 767;
-  const countryNames = production_countries?.map(country => (isMobile ? country.iso_3166_1 : country.name)) || [];
-  const releaseDate = new Date(release_date);
-  const year = releaseDate.getFullYear().toString();
+export const Tile = ({
+	poster_path,
+	title,
+	release_date,
+	production_countries,
+	vote_average,
+	votes,
+	overview,
+	genres
+}) => {
+	const isMobile = window.innerWidth < 767;
+	const countryNames =
+		production_countries?.map((country) =>
+			isMobile ? country.iso_3166_1 : country.name
+		) || [];
+	const releaseDate = new Date(release_date);
+	const year = releaseDate.getFullYear().toString();
 
-  return (
-    <Container>
-      <Image src={`https://image.tmdb.org/t/p/w500/${poster_path}`} alt="" />
-      <Content>
-        <Header>{title}</Header>
-        <Year>{year}</Year>
-        <Release>
-          <GreyRelease>Production:</GreyRelease>
-          {countryNames.toString().replace(",", ', ')}
-        </Release>
-        <ReleaseDate>
-          <GreyRelease>Release date:</GreyRelease>
-          {release_date}
-        </ReleaseDate>
-        <Genres>
-          {genres && genres.map(genre => (
-            <Genre key={genre.id}>{genre.name}</Genre>
-          ))}
-        </Genres>
-        <Rating>
-          <IconStar />
-          <ContainerRateText>
-            {vote_average ? vote_average.toFixed(2).toString().replace(".", ',') : "No votes yet"}
-            <Rate>/ 10</Rate>
-            <Votes>{votes}&nbsp;votes</Votes>
-          </ContainerRateText>
-        </Rating>
-      </Content>
-      <Discription>
-        {overview}
-      </Discription>
-    </Container>
-  );
+	return (
+		<Container>
+			<ImageWrapper>
+				<Image
+					src={`https://image.tmdb.org/t/p/w500/${poster_path}`}
+					alt=""
+				/>
+			</ImageWrapper>
+			<Content>
+				<Title>{title}</Title>
+				<Year>{year}</Year>
+				<Release>
+					<Production>{countryNames.toString().replace(",", ", ")}</Production>
+					<ReleaseDate>{release_date.toString().replaceAll("-", ".")}</ReleaseDate>
```

Comment:
It's great that you noticed it, but we also need to change the format. For example using release_date.toString().split("-").reverse().join(".")  I'll fix it, it's my mistake😃
<img width="288" alt="Zrzut ekranu 2023-11-25 o 02 42 06" src="https://github.com/jakub-bartoszek-developer/Movie-browser/assets/131645596/5c978180-c18f-4d2a-a480-53ce99d85f93">
<img width="424" alt="Zrzut ekranu 2023-11-25 o 02 42 19" src="https://github.com/jakub-bartoszek-developer/Movie-browser/assets/131645596/86fd090a-8919-4d61-b7ae-7a28d6743979">